app/services/unwithdraw_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class UnwithdrawService
+  def process(edition, user = nil)
```

Comment:
If this is just a generic "do your thing" method then going with "call" is probably the most consistent one as that has a special meaning in Ruby and is used in quite a lot of our single purpose services.